features/nanostack/FEATURE_NANOSTACK/mbed-mesh-api/mbed_lib.json

```diff
@@ -46,7 +46,7 @@
             "value": "NET_6LOWPAN_ROUTER"
         },
         "thread-pskd": {
-            "help": "Human-scaled commissioning credentials (string 6-255 chars).",
+            "help": "Human-scaled commissioning credentials. Uppercase alphanumeric string (0-9, A-Y excluding I, O, Q and Z), 6-32 characters).",
```

Comment:
Sorry to be that guy, but you have an extra `)` at the end of the sentence.
Once it's gone, I'll start CI. Feel free to squash your commits as well.

Reply:
Oops. Will keep as 2 commits, as first is just transplanting existing help text from other places, the second is new edits.